src/main/java/de/retest/recheck/review/workers/LoadFilterWorker.java

```diff
@@ -18,13 +20,30 @@
 public class LoadFilterWorker {
 
 	private final Counter counter;
+	private final Optional<Path> ignoreFile;
```

Comment:
I know that this is a controversial topic, but I wouldn't use `Optional` in fields.

See Stuart Marks' (who works for the core libraries team in the JDK group at Oracle) talk "Optional – The Mother of All Bikesheds" from Devoxx 2016 at [54:04](https://www.youtube.com/watch?v=Ej0sss6cq14&feature=youtu.be&t=54m04s):

>Why Not Use Optional in Fields?
>
>* More a style issue than a correctness issue
>   * usually there's a better way to model absence of a value
>   * use of Optional in fields often arises from slavish desire to eliminate nullable fields
>   * remember, eliminating nulls isn't a goal of Optional
>* Using Optional in fields…
>   * creates another object for every field
>   * introduces a dependent load from memory on every field read
>   * clutters up your code
>   * to what benefit? ability to chain methods?